particles/Demo/Demo.recipes

```diff
@@ -2,4 +2,4 @@ import 'DemoData.manifest'
 //import 'RestaurantsDemo.recipes'
 //import 'ProductsDemo.recipe'
 import 'TVMazeDemo.recipes'
-import 'PipeDemo.recipe'
+//import 'PipeDemo.recipe'
```

Comment:
Deleted?

Reply:
Don't understand the question. The pipes code in web-shell is deprecated in favor of pipes-shell. Much of it not yet _deleted_ because I want to do that judiciously, and what is done here is just for survival.